File: examples/php/organisation/organisation_details.php
<?php

$request = new HttpRequest();
$request->setUrl('http://exenzo-nl.saas/api/v1/organisation/0nz5d3LV');
$request->setMethod(HTTP_METH_GET);

$request->setHeaders(array(
    'cache-control' => 'no-cache',
    'content-type' => 'application/json',
    'x-authorization' => 'ORGANISATION_API_TOKEN'
));

try {
    $response = $request->send();

    echo $response->getBody();
} catch (HttpException $ex) {
    echo $ex;
}

File: examples/python/organisation/organisation_details.py
import http.client

conn = http.client.HTTPSConnection("exenzo-nl.saas")

headers = {
    'x-authorization': "ORGANISATION_API_TOKEN",
    'content-type': "application/json",
    'cache-control': "no-cache",
    }

conn.request("GET", "/api/v1/organisation/{organisation_id}", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))